sources/packages/web/src/components/aest/students/modals/BypassRestrictionModal.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <v-form ref="bypassRestrictionForm">
+    <modal-dialog-base :showDialog="showDialog" title="Bypass restriction">
+      >
+      <template #content>
+        <h3 class="category-header-medium my-4">Bypass information</h3>
+        <error-summary :errors="bypassRestrictionForm.errors" />
+        <v-card class="pa-4 border">
```

Comment:
Please review the modals content. The rectangle line should be `<content-group>` instead of `v-card`.